python-mip-sklearn/Dockerfile

```diff
@@ -1,24 +1,42 @@
-FROM hbpmip/python-base-build:0.4.4
+FROM hbpmip/python-mip:0.4.0
+
+RUN apt-get update
+RUN apt-get install -y git
 
 COPY sklearn_to_pfa/requirements-dev.txt /requirements-dev.txt
 RUN pip install -r /requirements-dev.txt
 
 COPY requirements.txt /requirements.txt
 RUN pip install -r /requirements.txt
 
+RUN pip install git+https://github.com/Marigold/hadrian.git@light#subdirectory=titus
+
+COPY sklearn_to_pfa /sklearn_to_pfa
+RUN cd sklearn_to_pfa && python setup.py install
+
 # Unit tests for sklearn_to_pfa
 COPY sklearn_to_pfa/tests /src/tests/
 
 WORKDIR /src
 RUN pytest tests/
 
+
+
 FROM hbpmip/python-mip:0.4.0
 
 COPY requirements.txt /
 COPY docker-entrypoint.sh /
 
 RUN pip install -r /requirements.txt
 
+RUN apt-get update
+RUN apt-get install -y git
```

Comment:
Two operations and no cleanup, this increases needlessly the size of the Docker image